edivorce/apps/core/migrations/0030_guid_as_username.py

```diff
@@ -11,6 +11,7 @@ class Migration(migrations.Migration):
 
     operations = [
          migrations.RunSQL(
-            "UPDATE core_bceiduser SET username = user_guid"
+            "UPDATE core_bceiduser SET username = user_guid",
+             reverse_sql="UPDATE core_bceiduser SET user_guid = username"
         )
```

Comment:
What was done for this migration is adding the reverse code to make sure it can be unapplied smoothly if needed.
This looks like a data migration, so maybe the fields are same because this one was run already. 
If we need to delete it, we would need to unapply it and fix the 0031 migration to become the new 0030. But I'm not sure it's worth doing so. We can squash migrations later if thing get bulky.